### PR TITLE
Add a transition to the project page after its creation

### PR DESCRIPTION
**Pull request by Aliaksandrzaz**

### CODE REVIEW

**Comment by kwoon** [comment]
You should add slug field in UI. This field  should suggest you slug of name but you can change this field whatever you like.

File: js/src/features/project/components/ProjectCreateForm.tsx [hunk 1]
```diff
@@ -18,11 +19,15 @@ export const CreateProjectForm: React.FC<Props> = ({
...
-  useFormServerErrors(form, errors, ["name", "description"]);
+  useFormServerErrors(form, errors, ["name", "description", "slug"]);
 
   const onFinish = useCallback(
     (values: any) => {
-      create(values);
+      create({
+        description: values.description,
+        name: values.name,
+        slug: !!values.slug ? convertToSlug(values.slug) : convertToSlug(values.name)
```
Comment (kwoon) [inline js/src/features/project/components/ProjectCreateForm.tsx:29]: It is not perfect solution. I imagine the flow in the following way:
1. User is typing the name of project
2. In the same moment automatically filled slug field (like computed field)
3. If user wants to modify slug, he can do this. And if thereafter he wants to change the name, slug should not be changed, cause user already choose slug name. To facilitate the task you can use `touched` property of form field (https://ant.design/components/form/#FieldData)

**Comment by kwoon** [comment]
#89

File: js/src/features/project/components/ProjectCreateForm.tsx (hunk 1 above)
Comment (Aliaksandrzaz) [reply js/src/features/project/components/ProjectCreateForm.tsx]: Fix

**Pull request merged**